Bottom: 2790b2a79a83ed06d52563be6edd3832b6761764
Top:    7747346890163d88fd38640ce1dd39347f8a6504
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-15 23:32:33 +0100

Add some more missing sources to POTFILES.in


---

diff --git a/po/POTFILES.in b/po/POTFILES.in
index d3d3bbb..b664517 100644
--- a/po/POTFILES.in
+++ b/po/POTFILES.in
@@ -19,6 +19,7 @@ src/find.c
 src/gpcb-menu.res.in
 src/hid/batch/batch.c
 src/hid/common/actions.c
+src/hid/common/hidgl_package_vrml_y.c
 src/hid/common/hidnogui.c
 src/hid/gerber/gerber.c
 src/hid/gtk/ghid-cell-renderer-visibility.c
